homework-submission/react-hyf/react-hyf-client/src/components/week3/ProfilPicture/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './style.css';
+
+function ProfilPicture({ userValues }) {
+  return <img src={userValues[11]} alt={userValues[0]} />;

```

Comment:
Don't you receive an object from the api? Why are you accessing it using an index? :)